### PR TITLE
Ingestors

### PR DESCRIPTION
**Pull request by 0ssigeno**

### CODE REVIEW

**Comment by 0ssigeno** [comment]
TODO: docs

**Comment by 0ssigeno** [comment]
TODO celery beat scheduler

File: intel_owl/tasks.py [hunk 1]
```diff
@@ -30,6 +31,21 @@ def update_plugin(state, plugin_path):
     plugin.update()
 
 
+@shared_task(soft_time_limit=120)
```
Comment (mlodic) [inline intel_owl/tasks.py:34]: maybe this is too low. 300?

File: docs/source/Usage.md [hunk 2]
```diff
@@ -490,6 +491,35 @@ The following are all the keys that you can change without touching the source c
...
+- `description`: _same as analyzers_
+- `python_module`: _same as analyzers_ 
+- `disabled`: _same as analyzers_
+- `config`:
+  - `queue`: _same as analyzers_
+  - `soft_time_limit`: _same as analyzers_
+- `playbook_to_execute`: Playbook that will be used for every IOC retrieved from the ingestor
+- `schedule`: Crontab schedule of its execution
+
 ---
```
Comment (mlodic) [inline docs/source/Usage.md]: could you populate the "Contribute" part too regarding Ingestors? ty

**Review by mlodic — COMMENTED** [review]
I report you this error too from the Admin section after clicking on "Crontab". 

http://localhost/admin/django_celery_beat/crontabschedule/

```

Template error:
In template /usr/local/lib/python3.9/site-packages/django/contrib/admin/templates/admin/change_list.html, error at line 68
   
   58 :         {% block search %}{% search_form cl %}{% endblock %}
   59 :         {% block date_hierarchy %}{% if cl.date_hierarchy %}{% date_hierarchy cl %}{% endif %}{% endblock %}
   60 : 
   61 :         <form id="changelist-form" method="post"{% if cl.formset and cl.formset.is_multipart %} enctype="multipart/form-data"{% endif %} novalidate>{% csrf_token %}
   62 :         {% if cl.formset %}
   63 :           <div>{{ cl.formset.management_form }}</div>
   64 :         {% endif %}
   65 : 
   66 :         {% block result_list %}
   67 :           {% if action_form and actions_on_top and cl.show_admin_actions %}{% admin_actions %}{% endif %}
   68 :            {% result_list cl %} 
   69 :           {% if action_form and actions_on_bottom and cl.show_admin_actions %}{% admin_actions %}{% endif %}
   70 :         {% endblock %}
   71 :         {% block pagination %}{% pagination cl %}{% endblock %}
   72 :         </form>
   73 :       </div>
   74 :       {% block filters %}
   75 :         {% if cl.has_filters %}
   76 :           <div id="changelist-filter">
   77 :             <h2>{% translate 'Filter' %}</h2>
   78 :             {% if cl.has_active_filters %}<h3 id="changelist-filter-clear">


Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/db/models/options.py", line 669, in get_field
    return self.fields_map[field_name]

During handling of the above exception ('human_readable'), another exception occurred:
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/utils.py", line 271, in lookup_field
    f = _get_non_gfk_field(opts, name)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/utils.py", line 302, in _get_non_gfk_field
    field = opts.get_field(name)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/options.py", line 671, in get_field
    raise FieldDoesNotExist(

During handling of the above exception (CrontabSchedule has no field named 'human_readable'), another exception occurred:
  File "/usr/local/lib/python3.9/site-packages/cron_descriptor/ExpressionDescriptor.py", line 120, in get_full_description
    day_of_week_desc = self.get_day_of_week_description()
  File "/usr/local/lib/python3.9/site-packages/cron_descriptor/ExpressionDescriptor.py", line 330, in get_day_of_week_description
    return self.get_segment_description(
  File "/usr/local/lib/python3.9/site-packages/cron_descriptor/ExpressionDescriptor.py", line 503, in get_segment_description
    description_content += get_single_item_description(segment)
  File "/usr/local/lib/python3.9/site-packages/cron_descriptor/ExpressionDescriptor.py", line 333, in <lambda>
    lambda s: get_day_name(s),
  File "/usr/local/lib/python3.9/site-packages/cron_descriptor/ExpressionDescriptor.py", line 303, in get_day_name
    return ExpressionDescriptor.number_to_day(int(exp))

During handling of the above exception (invalid literal for int() with base 10: '[2'), another exception occurred:
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
  File "/usr/local/lib/python3.9/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python3.9/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "/usr/local/lib/python3.9/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 966, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/templatetags/base.py", line 45, in render
    return super().render(context)
  File "/usr/local/lib/python3.9/site-packages/django/template/library.py", line 258, in render
    _dict = self.func(*resolved_args, **resolved_kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/templatetags/admin_list.py", line 336, in result_list
    "results": list(results(cl)),
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/templatetags/admin_list.py", line 312, in results
    yield ResultList(None, items_for_result(cl, res, None))
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/templatetags/admin_list.py", line 303, in __init__
    super().__init__(*items)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/templatetags/admin_list.py", line 213, in items_for_result
    f, attr, value = lookup_field(field_name, result, cl.model_admin)
  File "/usr/local/lib/python3.9/site-packages/django/contrib/admin/utils.py", line 282, in lookup_field
    attr = getattr(obj, name)
  File "/usr/local/lib/python3.9/site-packages/django_celery_beat/models.py", line 319, in human_readable
    human_readable = get_description('{} {} {} {} {}'.format(
  File "/usr/local/lib/python3.9/site-packages/cron_descriptor/ExpressionDescriptor.py", line 652, in get_description
    return descripter.get_description(DescriptionTypeEnum.FULL)
  File "/usr/local/lib/python3.9/site-packages/cron_descriptor/ExpressionDescriptor.py", line 104, in get_description
    return choices.get(description_type, self.get_seconds_description)()
  File "/usr/local/lib/python3.9/site-packages/cron_descriptor/ExpressionDescriptor.py", line 136, in get_full_description
    raise FormatException(description)

Exception Type: FormatException at /admin/django_celery_beat/crontabschedule/
Exception Value: 
```

**Review by mlodic — APPROVED** [review]

**Pull request merged**